llvm/lib/Passes/PassRegistry.def

```diff
@@ -573,6 +572,12 @@ FUNCTION_PASS_WITH_PARAMS(
       return StackLifetimePrinterPass(dbgs(), Type);
     },
     parseStackLifetimeOptions, "may;must")
+FUNCTION_PASS_WITH_PARAMS(
+    "scalarizer", "ScalarizerPass",
+    [](ScalarizerPassOptions Opts) { return ScalarizerPass(Opts); },
+    parseScalarizerOptions,
+    "load-store;no-load-store;variable-insert-extract;"
+    "no-variable-insert-extract;no-min-bits=N;min-bits=N;")
```

Comment:
`no-min-bits` isn't a thing if I'm reading above correctly

Reply:
I copied this from some of the other thresholds, but I don't see how it makes any sense. I think I'll just drop it